scapy/contrib/cansocket_python_can.py

```diff
@@ -165,6 +209,7 @@ def send(self, x):
                           arbitration_id=x.identifier,
                           dlc=x.length,
                           data=bytes(x)[8:])
+        msg.timestamp = time.time()
         try:
             x.sent_time = time.time()
```

Comment:
If I were you I'd do
`x.sent_time = msg.timestamp`
to have some consistency

Reply:
damn... I will fix it in #3075